internal/pkg/model/mapper.go

```diff
@@ -13,6 +16,91 @@ type MapperContext struct {
 	State  *State
 }
 
+type ObjectFiles struct {
```

Comment:
Novy struktura `ObjectFiles`. Ktora sa bude v dalsom PR pouzivat pri local/remote load.

Reply:
Teraz mame pre kazdy subor samostatnu polozku v `*Recipe`:
![image](https://user-images.githubusercontent.com/19371734/144796222-660f5595-ac73-41bc-a322-9b10e442eb17.png)

+ este `ExtraFiles`

Na to aby som mohol spravit so vsetkych JSON suborov sablony potrebujem mat vsetky subory v jednom field `Files` (dalsi PR).